src/chainparams.cpp

```diff
@@ -60,8 +60,8 @@ class CMainParams : public CChainParams {
         // Then, if miner consensus: 8MB max, doubling every two years
         consensus.nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
         consensus.nSizeDoubleEpoch = 60*60*24*365*2; // two years
-        consensus.nMaxSizeBase = 8*1000*1000; // 8MB
-        consensus.nMaxSizeDoublings = 10;
+        consensus.nMaxSizeBase = 2*1000*1000; // 2MB
+        consensus.nMaxSizeDoublings = 2;
```

Comment:
I suggest leaving this at nMaxSizeDoublings = 10;

It wouldn't make a difference until sometime after 2020, and prevents us from being in same place in the same situation again in 4 year where we need a hard fork to keep growing.